src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -207,26 +207,913 @@
+                                                <Run x:Uid="General_LastSettingsRestoreDate" />
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="0" Grid.Column="3">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run Text="{Binding LastSettingsRestoreDate, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+
+                                <Border Grid.Row="2" Grid.Column="0">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_LastSettingsBackupSource" />
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="2" Grid.Column="1">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run Text="{Binding LastSettingsBackupSource, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="2" Grid.Column="2" BorderThickness="1, 0, 0, 0" BorderBrush="Black">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run x:Uid="General_CurrentSettingMatchStatus" />
+                                    </TextBlock>
+                                </Border>
+
+                                <Border Grid.Row="2" Grid.Column="3">
+                                    <TextBlock Style="{StaticResource SecondaryTextStyle}">
+                                                <Run Text="{Binding CurrentSettingMatchText, Mode=OneWay}"/>
+                                    </TextBlock>
+                                </Border>
+
+                            </Grid>
+
+
+
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+
                 </controls:SettingsGroup>
+
+
+                <!-- SettingsBackupWasSuccessful -->
+                <InfoBar x:Uid="General_SettingsBackupMessageResults"
+                                  IsClosable="False"
+                                  Severity="Success"
+                                  IsTabStop="{Binding SettingsBackupWasSuccessful, Mode=OneWay}"
+                                  IsOpen="{Binding SettingsBackupWasSuccessful, Mode=OneWay}"
+                                  Message="{Binding SettingsBackupMessage}"/>
+
+                <!-- SettingsBackupWasUnsuccessful  -->
+                <InfoBar x:Uid="General_SettingsBackupMessageResults"
+                                  IsClosable="False"
```

Comment:
If we have an error we should use `severity=error`.

Reply:
OK, that's fine with me.

Reply:
On second thought, its not really an error... It's really just a warning because usually it's just a warning that backup or restore was not done because it did not need to be done.

In fact it's usually closer to Severity="Informational".  

I think I really need to refactor this to one one InfoBar with a dynamic Severity.